### PR TITLE
WE-1648 | NAS Switch

### PR DESCRIPTION
**Pull request by adam-young-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

## Checklist

- [ ] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Review by jason-dantzer-cko — APPROVED** [review]
LGTM

File: vercel.json [hunk 1]
```diff
@@ -1,3 +1,10 @@
 {
-  "trailingSlash": true
-}
+  "trailingSlash": true,
+  "redirects": [
```
Comment (tania-asimionesei-cko) [inline vercel.json:3]: Thanks for adding the redirect

File: web/index.html [hunk 2]
```diff
@@ -28,9 +33,8 @@
 		<script type="text/javascript">
 			function OptanonWrapper() {}
 		</script>
-
-		<script async type="text/plain" src="https://www.googletagmanager.com/gtag/js?id=UA-83886670-8"></script>
-		<script defer type="text/plain" class="optanon-category-C0002" src="../../js/gtm.js"></script>
+		<script async tyoe="text/plain" src="https://www.googletagmanager.com/gtag/js?id=UA-83886670-8"></script>
```
Comment (tania-asimionesei-cko) [inline web/index.html:36]: tyoe="text/plain" has a typo

File: web/index.html [hunk 3]
```diff
@@ -93,8 +106,6 @@
 			);
 		</script>
 
-		<script src="js/show-footer.js"></script>
```
Comment (tania-asimionesei-cko) [inline web/index.html:96]: We don't need the footer anymore?

**Review by tania-asimionesei-cko — CHANGES_REQUESTED** [review]
Just a couple of small things that I've spotted in the single comments. 
Everything else looks good thanks @adam-young-cko

**Comment by adam-young-cko** [comment]
@tania-asimionesei-cko Scripts should be fixed, and adjusted the footer logic slightly as it was overlapping the content. All should be rendering as it should now

**Review by jack-hutchinson-cko — DISMISSED** [review]
LGTM

**Comment by adam-young-cko** [comment]
@tania-asimionesei-cko Footer should now be "fixed" (it still overlaps but it's at least wrapping like we mentioned)

**Review by tania-asimionesei-cko — CHANGES_REQUESTED** [review]
Thanks a lot @adam-young-cko . 
Just one small thing that somehow we all missed, the URL for the banner, is pointing to the main tech docs instead of the previous API ref:
image

**Review by tania-asimionesei-cko — APPROVED** [review]
Thanks @adam-young-cko

**Review by chrisi-webster-cko — APPROVED** [review]

**Pull request merged**